src/operator/c_lapack_api.h

```diff
@@ -271,6 +339,9 @@ inline void flip<cpu, double>(int m, int n,
      " Ensure that lapack library is installed and build with USE_LAPACK=1 to get lapack" \
      " functionalities.")
 
+  #define MXNET_LAPACK_ROW_MAJOR 101
```

Comment:
I never really understood what these numbers (101, 102) mean. This is pure internal, right?

Reply:
yep